src/drive/web/modules/views/Folder/FolderViewBody.jsx

```diff
@@ -53,12 +79,16 @@ const FolderViewBody = ({
       const isNote = models.file.isNote(file)
       const isShortcut = models.file.isShortcut(file)
 
-      //Should only be called if mobile
       if (isShortcut) {
-        const resp = await client.query(
-          Q('io.cozy.files.shortcuts').getById(file.id)
-        )
-        window.location.href = resp.data.attributes.url
+        if (isMobileApp()) {
+          const resp = await client.query(
+            Q('io.cozy.files.shortcuts').getById(file.id)
+          )
+          window.location.href = resp.data.attributes.url
+        } else {
+          const url = generateFileUrl({ file, client, isFlatDomain: true })
+          window.open(url, '_blank')
+        }
```

Comment:
We introduced a regression in this commit, no? If we're renaming a file and we click on it, it'll follow the link, no?